src/js/controller/model.js

```diff
@@ -295,28 +295,28 @@ define([
             _currentProvider = null;
         };
 
-        this.setVolume = function(vol) {
-            vol = Math.round(vol);
-            _this.set('volume', vol);
+        this.setVolume = function(volume) {
+            volume = Math.round(volume);
+            this.set('volume', volume);
             if (_provider) {
-                _provider.volume(vol);
+                _provider.volume(volume);
             }
-            var muted = (vol === 0);
-            if (muted !== _this.get('mute')) {
-                _this.setMute(muted);
+            var mute = (volume === 0);
+            if (mute !== this.get('mute')) {
```

Comment:
Not really. We check for a change in `set` so that "change:" events are only sent when the value changes https://github.com/jwplayer/jwplayer/blob/master/src/js/utils/simplemodel.js